libraries/ts-command-line/src/test/__snapshots__/CommandLineParameter.test.ts.snap

```diff
@@ -3,46 +3,27 @@
 exports[`CommandLineParameter allows an undocumented synonym 1`] = `
 Array [
   "### --choice output: ###",
-  "--choice",
-  "one",
   "### --choice-with-default output: ###",
   "--choice-with-default",
-  "two",
+  "default",
```

Comment:
Huh, are these snapshot changes expected? I guess the tests were going wrong... interesting

Reply:
Yeah this was happening because `process.env` had values from a previous test